package.json

```diff
@@ -35,12 +36,15 @@
     "grunt-contrib-copy": "1.0.0",
     "grunt-contrib-jasmine": "1.0.3",
     "grunt-contrib-watch": "1.0.0",
-    "grunt-rollup": "1.0.1",
+    "grunt-rollup": "6.0.0",
     "grunt-template-jasmine-istanbul": "0.4.0",
     "jquery": "2.1.4",
+    "protobufjs": "6.8.0",
```

Comment:
Yes, userspace will be required to import it in their scope and provide as openapi factory configuration. It still present as dev dependency because it's used in unit tests.